src/refda/adm/ietf_dtnma_agent.c

```diff
@@ -377,33 +519,6 @@ static void refda_adm_ietf_dtnma_agent_edd_num_msg_tx(refda_edd_prod_ctx_t *ctx)
      */
 }
 
-// FUTURE:
```

Comment:
No objections to removing this, started a PR to incorporate this EDD and possibly others if it makes sense: https://github.com/JHUAPL-DTNMA/dtnma-adms/pull/9